flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/sessionjob/FlinkSessionJobReconciler.java

```diff
@@ -76,6 +78,24 @@ public void reconcile(FlinkSessionJob flinkSessionJob, Context context) throws E
                 OperatorUtils.getSecondaryResource(
                         flinkSessionJob, context, configManager.getOperatorConfiguration());
 
+        var labels = flinkSessionJob.getMetadata().getLabels();
```

Comment:
I suggest to extract this into a utility method (we could use computeIfAbsent)

Reply:
Moved to a dedicated method.